rudder-core/src/main/scala/com/normation/rudder/reports/execution/ReportsExecutionService.scala

```diff
@@ -149,6 +160,7 @@ class ReportsExecutionService (
         reportsRepository.getReportsWithLowestId match {
           case Full(Some((id, report))) =>
             logger.debug(s"Initializing the status execution update to  id ${id}, date ${report.executionTimestamp}")
+            this.idForCheck = id
```

Comment:
the idForCheck ? For init, else it remains at 0 (it would change on next run, but I don't think it's a problem ?)